learn/configuration/instance_options.md

```diff
@@ -46,6 +137,15 @@ Environment variables are always identical to the corresponding command-line opt
 
 ## All instance options
 
+### Configuration file
```

Comment:
This option cannot be used in the config file. Maybe could we mention it here again?

Reply:
Yes, good idea!